content/authorization.md

```diff
@@ -1054,15 +1054,30 @@ preceding documentation.
   such devices to participate in the authorization
   ecosystem.
 
+- What happens when a user revokes my application's
+  access to their data?
+
```

Comment:
I am guessing that by having this section, it's implied that there is a way to revoke the refresh token for an app. Shall we mention that the operation can be done by an administrator currently?

Reply:
It *can't* be done by an administrator currently (only by the users themselves).